Cargo.toml

```
@@ -4,16 +4,16 @@ description = "Cookie and session middleware for conduit-based stacks"
 license = "MIT"
 name = "conduit-cookie"
 repository = "https://github.com/conduit-rust/conduit-cookie"
-version = "0.8.6"
+version = "0.9.0-alpha.0"
 
 [dependencies]
 base64 = "0.11"
-conduit = "0.8"
-conduit-middleware = "0.8"
+conduit = "0.9.0-alpha.0"
+conduit-middleware = "0.9.0-alpha.0"
 
 [dependencies.cookie]
 features = ["secure"]
 version = "0.11"
```

```suggestion
version = "0.12"
```

We can use v0.12 since it also uses time v0.1 and using it can be one step to remove old base64.

---

Sounds good to me.  Thanks!